### PR TITLE
Add recipe for MPDAF

### PR DESCRIPTION
**Pull request by saimn**
So this is the reason why I needed the fitsio and cfitsio packages ;-)

### CODE REVIEW

File: recipe_templates/mpdaf/build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
```
Comment (saimn) [inline recipe_templates/mpdaf/build.sh:3]: Not sure what is the best way to be able to use the cfitsio package, this seems fine but maybe there is a better way ?

Comment (mwcraig) [inline recipe_templates/mpdaf/build.sh:3]: I'm actually not sure, but if this works, go with it.

Comment (saimn) [inline recipe_templates/mpdaf/build.sh:3]: Ok, it works so let's go with it. I pushed a fix to remove the version-dependent url, then it should be fine.

**Comment by mwcraig** [comment]
Merging, thanks!

**Comment by saimn** [comment]
Thanks !

**Pull request merged**